test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/ReleaseCaseControllerSpec.scala

```diff
@@ -121,7 +121,7 @@ class ReleaseCaseControllerSpec extends WordSpec with Matchers with UnitSpec
     "return OK and HTML content type" in {
       when(casesService.getOne(refEq("reference"))(any[HeaderCarrier])).thenReturn(successful(Some(caseWithStatusNEW)))
       when(queueService.getOneBySlug("queue")).thenReturn(Some(queue))
-      when(casesService.releaseCase(refEq(caseWithStatusNEW), any[Queue])(any[HeaderCarrier])).thenReturn(successful(caseWithStatusOPEN))
+      when(casesService.releaseCase(refEq(caseWithStatusNEW), any[Queue])(any[HeaderCarrier], any[Operator])).thenReturn(successful(caseWithStatusOPEN))
```

Comment:
I would use a real string with `refEq` for the `operator` param.
This strictness will give the advantage of failing the tests if we introduce some bugs in the code later in the future.

Reply:
Sorted